### DISCUSSION TITLE
Design

### DISCUSSION THREAD

**Discussion by koustakken** [bandinopla/weightxreps-client#26] [Ideas]
When I talked about the possibilities of changing the ui, I meant something like this

![image_2024-05-17_12-42-53](https://github.com/bandinopla/weightxreps-client/assets/73124280/2afe413b-d5cd-485b-8e4e-a63dcc75de3a)

If you are interested, I could make a test layout for different sections of the application

**Comment by bandinopla** [comment]
as it is, in that screenshot, I feel it lacks more visual impact, it looks too plain. Also, are you taking into account the mobile adaptation?
